onnx_cntk/tests/onnx_backend_test.py

```diff
@@ -0,0 +1,50 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+import onnx.backend.test
+
+from onnx_cntk import backend as cntk_backend
+
+# This is a pytest magic variable to load extra plugins
+pytest_plugins = 'onnx.backend.test.report'
+
+backend_test = onnx.backend.test.BackendTest(cntk_backend, __name__)
+
+
+skip_models = [
+    'bvlc_alexnet',
+    'densenet121',
+    'inception_v1',
+    'inception_v2',
+    'resnet50',
+    'shufflenet',
+    'vgg16',
+    'vgg19',
+]
+
+skip_ops = [
+    'test_max',
+    'test_min',
+    'BatchNorm',
```

Comment:
Just curious -why are these ops skipped?

Reply:
I'll remove 'BatchNorm' from this list. But the other two ops cause crashes during test report generation. We can look at them separately to see why, but for now we can skip them to unblock test report generation.

Reply:
OK.